demos/demo_raw_coordinate_system.py

```diff
@@ -0,0 +1,128 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+
+import pykitti
+
+basedir = 'raw'
+date = '2011_09_26'
+drive = '0001'
+
+data = pykitti.raw(basedir, date, drive)
+
+"""
+The coordinate systems of pointcloud:
+
+* ImageX:     Each point's corresponding pixel coordinate. eg. z_i = [u,v,1]
+* CameraX:    Pointcloud coordinate at four cameras (0,1,2,3). eg. y_i = [x,y,z,1]
+* Reference:  Equal to Camera0. eg. y_0 = [x,y,z,1]
+* Rectified:  Aligned coordinate for the four cameras. eg. r = [x,y,z,1]
+* Velodyne:   Original velodyne coordinate. eg. v = [x,y,z,r]
+* IMU/GPS:    Pointcloud coordinate at the IMU device. eg. m = [x,y,z,1]
+* World:      Real world coordinate, eg. w = [x,y,z,1]
+
+Pykitti transformation matrices:
+
+data.calib.T_camX_velo:  Velodyne -> CameraX
+data.calib.K_camX:       CameraX -> ImageX
+data.calib.T_velo_imu:   Velodyne -> IMU

```

Comment:
Hi, thank you very much for providing this useful coordinate system transformation code.

I noticed that you directly used data.calib.T_velo_imu in the matrix multiplications to get from velodyne -> IMU coordinate system in line 117. However, from my understanding, T_velo_imu is the transformation matrix to get from IMU -> Velodyne coordinate system. As with all other transformation matrices, the general format is T_target_source.

I believe that is also the reason why #17 is also brought up. 

Do correct me if my understanding is wrong!